<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.12.3 on darwin)" generated="2024-08-19T20:19:33.582751" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="/Users/oleksandr.chako/.vscode/extensions/robocorp.robotframework-lsp-1.12.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-08-19T20:19:33.595269" elapsed="3.738451"/>
</kw>
<status status="PASS" start="2024-08-19T20:19:33.595097" elapsed="3.739196"/>
</test>
<status status="PASS" start="2024-08-19T20:19:33.583608" elapsed="3.753285"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-08-19T20:19:33.577917" level="WARN">Error in file '/Users/oleksandr.chako/.vscode/extensions/robocorp.robotframework-lsp-1.12.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
